ads/doubleclick.js

```diff
@@ -60,9 +60,14 @@ export function doubleclick(global, data) {
       }
 
       pubads.addEventListener('slotRenderEnded', function(event) {
+        let creativeId = event.creativeId ||
```

Comment:
It better be, since there is an assert down the call path. The answer is yes because of line 70 :)